README.md

```diff
@@ -0,0 +1,14 @@
+# Cloud IoT API
+
+[![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/iot.svg)](https://pkg.go.dev/cloud.google.com/go/iot)
+
+Go Client Library for ClearBlade IoT Core API.
+
+## Authorization
+
+See the [Authorization](https://clearblade.atlassian.net/wiki/spaces/IC/pages/2240675843/Add+service+accounts+to+a+project)
```

Comment:
Are there any docs for the CLEARBLADE_CONFIGURATION env variable? Here's how we call it out in the node readme - https://github.com/ClearBlade/nodejs-iot/#setting-up-service-account-credentials